AdlsSDKGettingStarted/Program.cs

```diff
@@ -12,99 +13,85 @@ public class Program
         private static string applicationId = "FILL-IN-HERE";     // Also called client id
         private static string clientSecret = "FILL-IN-HERE";
         private static string tenantId = "FILL-IN-HERE";
-        private static string adlsAccountFQDN = "FILL-IN-HERE";   // full account FQDN, not just the account name like example.azure.datalakestore.net
+        private static string serviceUri = "FILL-IN-HERE";        // full account FQDN, not just the account name like https://{ACCOUNTNAME}.dfs.core.windows.net/
```

Comment:
This reads a little confusingly.
```suggestion
        private static string serviceUri = "FILL-IN-HERE";        // full account FQDN, not just the account name - it should look like https://{ACCOUNTNAME}.dfs.core.windows.net/
```